### ISSUE TITLE
Summer of Bitcoin project proposal

### ISSUE THREAD

**Issue by thunderbiscuit** [bitcoindevkit/bdk-ffi#101]
I have developed my own issues with the project proposals I hope to get applicants for through Summer of Bitcoin, but one of them is actually on helping develop and test bdk-python. The [proposal is here](https://github.com/thunderbiscuit/padawan-wallet/issues/151).

I just realized that it might be more discoverable to have it here instead. Any thoughts? If I decide to take a student on this I would obviously take the lead on it, but I'm just wondering if you'd be opposed to me moving the issue here so it lives with the others.

**Comment by notmandatory** [comment]
Yes let's move it and put your name as the mentor of course.

**Comment by thunderbiscuit** [comment]
Yes I tried to move it myself but you can only move between repositories you own. So I just opened a new issue here. I also let Adi know.
